anthos-bm-gcp-terraform/main.tf

```diff
@@ -277,6 +279,20 @@ resource "local_file" "cluster_yaml_manuallb" {
   })
 }
 
+// Generate the Anthos bare metal nfs yaml file using the template. This
+// resource is created only if the nfs_server is true.
+resource "local_file" "nfs_yaml" {
```

Comment:
How is it guaranteed that this get's created only if `nfs_server == true`?
It seems like the file will still be created but with empty values for `nfs_server` and `nfs_share`.

Reply:
Unfortunately correct, see this discussion for more detail: https://github.com/GoogleCloudPlatform/anthos-samples/pull/258#discussion_r861898037

Reply:
Ahhh I see. Can we add an additional comment
```
// Generate the Anthos bare metal nfs yaml file using the template.
// This file is only used when the `nfs_server` variable is set to true.
```